spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowSuspiciousConnectsModel.scala

```diff
@@ -199,6 +199,16 @@ object FlowSuspiciousConnectsModel {
     val dataWithWords = totalRecords.withColumn(SourceWord, flowWordCreator.srcWordUDF(ModelColumns: _*))
       .withColumn(DestinationWord, flowWordCreator.dstWordUDF(ModelColumns: _*))
 
+    dataWithWords.cache()
```

Comment:
Caching dataWithWords helps performance as it's used in 2 jobs, one to get corrupt records (error on word creation, flagged as word_error) and then it's primary function which is document word count. 
Cached ensures dataWithWords is not re-calculated when it's materialized in SpotLDAWrapper.